### PR TITLE
front end Checkout and StripeCheckoutButton component

### PR DESCRIPTION
**Pull request by robertpengcode**
_Your PR Notes Here_

built StripeCheckoutButton component using react-stripe-checkout. 
built Checkout component to contain StripeCheckoutButton (shown as "Pay Now" button).

can see checkout page at path "/checkout" (so far only one button)
when click button can see the payment form now.

no back end route built yet, no front end exios post call function built yet

### CODE REVIEW

File: client/components/productDetail.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ class ProductDetail extends Component {
       return <h1>loading...</h1>
     } else {
       let product = this.props.product
+      console.log(product.reviews)
```
Comment (avc278) [inline client/components/productDetail.js:41]: let's remove any console logs before merging to master

File: client/components/index.js [hunk 2]
```diff
@@ -15,3 +15,5 @@ export {Login, Signup} from './auth-form'
 export {default as ProductCard} from './productCard'
 export {default as ProductDetail} from './productDetail'
 export {default as Products} from './products'
+export {default as Checkout} from './checkout'
+export {default as StripeCheckoutButton} from './stripeCheckoutButton'
```
Comment (avc278) [inline client/components/index.js:19]: alphabetize these

File: client/components/stripeCheckoutButton.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react'
+import StripeCheckout from 'react-stripe-checkout'
+
+const StripeCheckoutButton = ({price}) => {
+  const priceForStripe = price * 100
+  const publishablekey = 'pk_test_fuzmrsq8tsZrctgTeJmHxClb00cwD5jxSX'
```
Comment (avc278) [inline client/components/stripeCheckoutButton.js:6]: According to the stripe API documentation, we don't have to hide this API key, so this looks good.

File: client/components/stripeCheckoutButton.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  const onToken = token => {
+    console.log(token)
```
Comment (avc278) [inline client/components/stripeCheckoutButton.js:9]: remove this

Comment (robertpengcode) [reply client/components/stripeCheckoutButton.js:9]: yes, removed

File: client/components/productDetail.js (hunk 1 above)
Comment (robertpengcode) [reply client/components/productDetail.js:41]: removed

File: client/components/stripeCheckoutButton.js (hunk 3 above)
Comment (robertpengcode) [reply client/components/stripeCheckoutButton.js:6]: yeah, it may go to .env after backend is built

File: client/components/index.js (hunk 2 above)
Comment (robertpengcode) [reply client/components/index.js]: yes

**Pull request merged**